app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
```

Comment:
Use double quotes for consistency in your code. The import statement should use double quotes as per the checklist.